sdk/communication/azure-communication-identity/src/test/resources/session-records/createAsyncIdentityClientUsingConnectionString.json

```diff
@@ -0,0 +1,26 @@
+{
+  "networkCallRecords" : [ {
+    "Method" : "POST",
+    "Uri" : "https://REDACTED.communication.azure.com/identities?api-version=2021-03-07",
+    "Headers" : {
+      "User-Agent" : "azsdk-java-UnknownName/UnknownVersion (11.0.8; Windows 10; 10.0)",
```

Comment:
why does it not detect the package name and version?

Reply:
Updated